components/authz-service/server/v2/system_integration_test.go

```diff
@@ -57,6 +59,103 @@ func TestIntegrationSystemPolicies(t *testing.T) {
 	}
 }
 
+// bug: this test passes contingent on the above test being run
```

Comment:
Help the reader parse the multi-line comment. 😁 
```suggestion
// bug: this test passes contingent on the above test being run.
```